internal/utils/config.go

```diff
@@ -83,13 +84,21 @@ func (s *sizeInBytes) UnmarshalText(text []byte) error {
 
 var Config config
 
+// We follow these rules when adding new config:
+//  1. Update init_config.toml with the new key, default value, and comments to explain usage.
+//  2. Update config struct with new field and toml tag, written out in snake_case.
+//  3. Add custom validations to LoadConfigFS function for the new field, such as range checks.
+//
+// If you are adding new secrets, such as API keys, use env var instead of toml. For example,
+//  1. Config.Auth.AnonKey is tagged with `toml:"-" mapstructure:"anon_key"`. This tag prevents
+//     externalising to toml but allows reading from an env var named SUPABASE_AUTH_ANON_KEY.
+//  2. Default values should be added to LoadConfigFS function, after checking for empty value.
 type (
 	config struct {
 		ProjectId string              `toml:"project_id"`
 		Api       api                 `toml:"api"`
-		Db        db                  `toml:"db" mapstructure:"db"`
+		Db        db                  `toml:"db"`
 		Studio    studio              `toml:"studio"`
-		Kong      kong                `toml:"kong"`
```

Comment:
This is a breaking change :( Is this already released on stable?

Reply:
Nah it's still on beta so I think should be ok

Reply:
I propose we take out this config and set it to 1, and if people want it to be reverted we can revisit adding this config.

Reply:
I think one use case for overriding the default value is CI runner. Users may have many typescript integration tests hitting the api in parallel. Setting it to a higher value could make their tests run faster.

Reply:
Sure, we can revisit this if/when a user complains that their tests are slow. Otherwise we risk adding a config that might become irrelevant in the future.

Consider e.g. when we replace Kong with Envoy/Edge Runtime - there may not be an equivalent config in those services.